streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -875,10 +876,11 @@ public void process(final String key, final Long value) {
         }
 
         private void flushStore() {
-            final KeyValueIterator<String, Long> it = store.all();
-            while (it.hasNext()) {
-                final KeyValue<String, Long> next = it.next();
-                context.forward(next.key, next.value);
+            try (final KeyValueIterator<String, Long> it = store.all()) {
```

Comment:
nice catch! Can we extract this into it's own PR so we can back port it to older branches?

Reply:
Done https://github.com/apache/kafka/pull/6562

Reply:
Thanks.